java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -96,16 +99,16 @@ public SymbolicValue eval(ProgramState programState, Tree syntaxNode) {
         }
         break;
       }
-      case LOGICAL_COMPLEMENT : {
+      case LOGICAL_COMPLEMENT: {
         UnaryExpressionTree unaryExpressionTree = (UnaryExpressionTree) syntaxNode;
         SymbolicValue val = eval(programState, unaryExpressionTree.expression());
-        if(SymbolicValue.FALSE_LITERAL.equals(val)) {
+        if (SymbolicValue.FALSE_LITERAL.equals(val)) {
           return SymbolicValue.TRUE_LITERAL;
-        } else if(val.equals(SymbolicValue.TRUE_LITERAL)) {
+        } else if (val.equals(SymbolicValue.TRUE_LITERAL)) {
           return SymbolicValue.FALSE_LITERAL;
         }
-        //if not tied to a concrete value, create symbolic value with no constraint for now.
-        //TODO : create constraint between expression and created symbolic value
+        // if not tied to a concrete value, create symbolic value with no constraint for now.
+        // TODO : create constraint between expression and created symbolic value
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)